### ISSUE TITLE
Tomodachi Life for some odd reason, is not creating save data

### ISSUE THREAD

**Issue by Jaedinator** [citra-emu/citra#7070]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Citra Nightly 2010

### Description of Issue

Hello, so basically I started Tomodachi Life today and when I opened up the game, It is only stuck on the "creating save data" and it will not let me continue. I have uninstalled Citra, I have used a different Tomodachi Life ROMS and yet they don't work... Is there a way do fix this? (Right now, I'm going to restart my PC and if it works, I will give a quick update)
Thank you :)

### Expected Behavior

![Screenshot 2023-10-14 131615](https://github.com/citra-emu/citra/assets/136248132/6ad7f000-3eb9-40dd-be01-252e6fab043e)

### Reproduction Steps

No idea

### Log File

[citra_log.txt](https://github.com/citra-emu/citra/files/12907067/citra_log.txt)

### System Configuration

CPU: unknown
GPU/Driver: unknown
RAM: 8.00 GB (7.65 GB usable)
OS: Windows 11

**Comment by Jaedinator** [comment]
Edit: All of my games (expect for animal crossing new leaf) Don't even work at all. I think it's a Citra problem in general. (I could be wrong though.)

**Comment by Jaedinator** [comment]
Edit 2: I reinstalled Citra and so far, nothing is working. I'm checking to see if I'm missing any files in my SDMC folder but I don't really know if anything is missing in there at all... And I don't know if my Citra folders are corrupted... I tried using Canary but the problem did not change. I will try and look for tutorials on how to fix this. I think I forgot to bring this up but originally, i used to have save data for all my games. but unfortunately, I tried to open Mii Maker but It keeps crashing, I thought it had something to do with my ROMS but it did not. And how the Mii Maker will not open was from me trying to scan QR codes for certain Miis but Citra crashed and it works fines when I reopen citra but when I clicked on Mii Maker it crashes Citra. (This has already been fixed because I restarted my Citra data and now nothing works, not even games will load. I think it has something to do with my log and I think I should replace it or delete it. Idk what to do...)
